### PR TITLE
zh-translation: docs/ops/configuration/traffic-management/protocol-se…

### PR DESCRIPTION
**Pull request by orangegzx**
…lection/index.md

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by kebe7jun — COMMENTED** [review]
我看英文文档的段落和这个不一致吧？
https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/

**Comment by orangegzx** [comment]
> 我看英文文档的段落和这个不一致吧？
> https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/

已修改

File: content/zh/docs/ops/configuration/traffic-management/protocol-selection/index.md [hunk 1]
```diff
@@ -1,38 +1,57 @@
...
+- `gRPC`
+- `gRPC-Web`
+- `Mongo`
+- `MySQL`\*
+- `Redis`\*
+- `UDP` (UDP 不会被代理，但可以将端口指定为 UDP)
 
-下面是一个 Service 例子，它定义了一个 `mysql` 端口 和一个 `http` 端口：
+\* 在默认情况下，这些协议是禁用的，目的是避免无意启用 Experimental Feature。
+如需启用它们，需配置相应的 Pilot [environment variables](/zh/docs/reference/commands/pilot-discovery/#envvars)。
```
Comment (kebe7jun) [inline content/zh/docs/ops/configuration/traffic-management/protocol-selection/index.md:52]:
```suggestion
如需启用它们，需配置相应的 Pilot 的[环境变量](/zh/docs/reference/commands/pilot-discovery/#envvars)。
```

**Review by kebe7jun — APPROVED** [review]
LGTM

**Pull request merged**